### ISSUE TITLE
Lord Rhyolith - Heroic Concussive Stomp Damage

### ISSUE THREAD

**Issue by clam119** [gamefreedomgit/Maelstrom#4348]
This will be a re-post of my response to the following report made by @Sketish - https://github.com/gamefreedomgit/Maelstrom/issues/4300 

Adding onto what Sketish has previously mentioned about Heroic Concussive Stomp damage, I've checked retail kills after the 14th July 2011, and the base damage of HC Rhyolith's Concussive Stomp is still doing 32,000 - 35,000 damage after the aforementioned "hotfix."

This first screenshot from November 10th 2011: 
![image](https://github.com/gamefreedomgit/Maelstrom/assets/42842394/0b686a15-da1f-4887-8743-bdefd18c192a)
 shows the Holy Paladin that only has Resistance Aura and no prior absorption shields, receives 31435 damage from Concussive Stomp. In addition to this, from the same encounter, in this second screenshot, again without absorption shields & defensive cooldowns, the Holy Paladin takes 34819 damage from Concussive Stomp: https://i.ibb.co/HLS66d1/image.png

The third screenshot from November 3rd 2011 in a separate guild's encounter: ![image](https://github.com/gamefreedomgit/Maelstrom/assets/42842394/207fed11-177c-476d-a1d6-ac5593caf5a3) shows the Balance Druid that also only has Resistance Aura and no prior absorption shields, receives a total of 32735 damage (25295 taken and 7440 resisted) from Concussive Stomp.

Lastly, the following images show a piece of conclusive evidence that the base 35,000 damage is in fact. The following screenshots are of the journals showing the damage for Concussive Stomp on both difficulties:

Normal Version: ![image](https://github.com/gamefreedomgit/Maelstrom/assets/42842394/598506cc-2129-4355-9e66-b96f4184cfc8) --> Which has the "hotfixed" heroic damage of 35,000. But, in reality, the Concussive Stomp damage on normal is significantly lower as shown here: https://i.ibb.co/tcJpy07/image.png where the Concussive Stomp deals 26632 with Fire Resistance active from the Shaman's Totem.
Heroic Version: ![image](https://github.com/gamefreedomgit/Maelstrom/assets/42842394/ccec5146-dea1-443c-ba40-6c517e18eb02) --> On the other hand, the value of Concussive Stomp on the journal is correct and reflects blizzlike numbers.
In conclusion, the "hotfix" on HC Lord Rhyolith's Concussive Stomp was so that the base damage is set to 35,000 before resistance & absorption, and that the value reflected in the journal are blizzlike numbers as evident in the "post-hotfix" changes to HC Concussive Stomp shown in the first-to-third screenshots.

P.S. Check the Normal & Heroic Version's damage numbers on Concussive Stomp to see that the Normal Version listed in the journal is incorrect.  If you take, for example, the Blackout damage on Normal & Heroic Valiona & Theralion, then you can see that the numbers are both different.

**Comment by Polynomix** [comment]
July 14
-Rhyolith : Concussive Stomp damage has been increased on Heroic difficulty.

September 20
-Lord Rhyolith : Health and damage have been reduced by 15% on both normal and Heroic difficulty.

Damage of the stomp between July 14th 2011 and September 20th 2011 was like 42k-47k. It was ~35k before and after

check first stomp  damage of these video of rhyolith during this time interval
https://www.youtube.com/watch?v=pjaXzRW1D2A
https://www.youtube.com/watch?v=wkAhz9JeyoU
